src/cheatcodes/pages/features/tutorial/CheatcodesNavigationProfileTutorial.tsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react'
 
 import { CheatcodesTemplateScreen } from 'cheatcodes/components/CheatcodesTemplateScreen'
-import { LinkToScreen } from 'cheatcodes/components/LinkToScreen'
+import { LinkToScreenWithNavigateTo } from 'cheatcodes/components/LinkToScreenWithNavigateTo'
+import { getOnboardingNavConfig } from 'features/navigation/OnboardingStackNavigator/getOnboardingNavConfig'
 
 export function CheatcodesNavigationProfileTutorial(): React.JSX.Element {
   return (
     <CheatcodesTemplateScreen title="ProfileTutorial 👤">
-      <LinkToScreen screen="ProfileTutorialAgeInformationCredit" title="Page 17-18 ans (V3)" />
+      <LinkToScreenWithNavigateTo
+        title="Page 17-18 ans (V3)"
```

Comment:
Maintenant je me dit qu'on peut faire ça : 
```suggestion
        title="ProfileTutorialAgeInformationCredit"
```